Chiazokam/Politico
# #163504422 create the get one party endpoint

#### What does this PR do?
This PR tests and creates the get one party Endpont

#### Description of Task to be completed?
- write tests for every test case
- implement admin check with middleware
- create the get one party endpoint
- modify db.sql to follow column name specs

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-get-one-party-163504422`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `GET` request to `http://localhost:3000/api/v1/parties/1`
- You should receive a successful response

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#163504422](https://www.pivotaltracker.com/story/show/163504422)

### Images
![image](https://user-images.githubusercontent.com/26940294/51907248-8bf38580-23c6-11e9-8638-4991da1b573a.png)
